src/components/CandidateBox.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Candidate from './Candidate';
+import axios from 'axios';
+import { url } from './url';
```

Comment:
이렇게 분리해서 정리해주시면 좋을 것 같아요 :)
```suggestion
import React, { useState, useEffect } from 'react';
import styled from 'styled-components';
import axios from 'axios';

import Candidate from './Candidate';
import { url } from './url';

```

Reply:
훨씬 보기 깔끔한 것 같네요 😄